### DISCUSSION TITLE
How can I disable the collaborative view?

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#8632] [Q&A]
How can I disable the collaborative view mode so that users are in personal mode, ensuring that their filter results do not affect other users? Currently, it seems there are only collaborative and locked view.

**Comment by dstala** [comment]
@c2006878 Personal views are not available at the moment. 
There is a feature request on that [Feature : Spreadsheet : Support Personal View](https://github.com/nocodb/nocodb/issues/144)
